eng/common/Invoke-WithRetry.ps1

```
@@ -5,11 +5,12 @@
 param (
         [Parameter(Mandatory = $true)][string]$Cmd,
         [int]$Retries = 2,
-        [int]$WaitFactor = 6
+        [int]$WaitFactor = 6,
+        [string]$errorActionPref = 'Stop'
```

Is this change necessary?  I'm not spotting anyplace that is making use of the new parameter, perhaps I overlooked it.

---

It was included to have an option to change the call to pull down the installation script (SilentlyContinue vs. Stop).

https://github.com/dotnet/dotnet-docker/blob/0b1c2706801908988ce79efcd5acd1042aa785db/tests/run-tests.ps1#L74

---

Why would you want to silently continue in this case?  If it is not being used with these changes, I don't the change should be made.

---

Sounds fine, we can remove the optional parameter here.

---

Sounds good, removing that as an optional parameter.